www/docs/getting-started/example.md

```diff
@@ -72,11 +69,12 @@ export default function Page() {
 You can use the `useSession` hook from anywhere in your application (e.g. in a header component).
 :::
 
-### Add session state
+### Share/configure session state
 
-To allow session state to be shared between pages - which improves performance, reduces network traffic and avoids component state changes while rendering - you can use the NextAuth.js Provider in `pages/_app.js`.
+Use the `<SessionProvider>` to allows instances of `useSession()` to share the session object across components. It also takes care of keeping the session updated and synced between tabs/windows.
```

Comment:
```suggestion
Use the `<SessionProvider>` to allow instances of `useSession()` to share the session data across components. `<SessionProvider>` will also take care of keeping the session updated and synced between tabs/windows.
```
I'm not sure about this though given is not possible anymore to use `useSession` without `<SessionProvider />` 🤔

Reply:
Yes, we should rephrase this to be explicit that users *must* use `SessionProvider` from now on.